docs/api.md

```diff
@@ -523,7 +523,7 @@ Disconnects Puppeteer from the browser, but leaves the Chromium process running.
 Promise which resolves to a new [Page] object. The [Page] is created in a default browser context.
 
 #### browser.pages()
-- returns: <[Promise]<[Array]<[Page]>>> Promise which resolves to an array of all open pages.
+- returns: <[Promise]<[Array]<[Page]>>> Promise which resolves to an array of all open pages. Non visible pages, such as `"background_page"` and `"service_worker"`, will not be listed here. You can find them using [target.page()](#targetpage).
```

Comment:
nit: service_worker isn't a page, so I think it isn't worth being mentioned here.

Reply:
done. 10x